packages/sentinel/src/models/category.ts

```diff
@@ -1,8 +1,13 @@
+import { NotificationType } from './notification';
+
 // copied from /models/src/types/notification-category.req.d.ts
 export interface SaveNotificationCategoryRequest {
   name: string;
   description?: string;
-  notificationIds: string[];
+  /**
+   * @maxItems 2
```

Comment:
why max 2 out of curiosity?

Reply:
Defined in the v0 scope. I'll DM you the slack conversation